travis/install.sh

```diff
@@ -86,7 +86,7 @@ VERBOSE=1 make -j${NPROC}
 make install
 # The cmake build is not able to generate the null file, so copy it at hand
 cp /tmp/proj_autoconf_install_from_dist_all/share/proj/null /tmp/proj_cmake_install/share/proj
```

Comment:
I believe that this cp and the above comment are no longer valid since data/null is now in git, and no longer generated